makefiles/info-global.inc.mk

```diff
@@ -0,0 +1,102 @@
+.PHONY: info-buildsizes info-buildsizes-diff info-features-missing \
+        info-boards-features-missing
+
+USEMODULE_GLOBAL := $(USEMODULE)
+USEPKG_GLOBAL := $(USEPKG)
+FEATURES_REQUIRED_GLOBAL := $(FEATURES_REQUIRED)
+FEATURES_OPTIONAL_GLOBAL := $(FEATURES_OPTIONAL)
+DISABLE_MODULE_GLOBAL := $(DISABLE_MODULE_GLOBAL)
+
+define board_missing_features
```

Comment:
indentation is off in this function

Reply:
Looks fine here, all tabs. anything specific?

Reply:
[Before](https://github.com/RIOT-OS/RIOT/pull/7589/files#diff-0acd95102a424d1a1c6e28cb4bfef379L261) it was not indented this way. That's why I pointed this out. I don't know what's the best practice here

Reply:
fixed it to two spaces